src/master/wpt_run_step.py

```diff
@@ -83,7 +83,10 @@ def makeWptRunCommand(properties):
             # Chrome to call getUserMedia without failing out.
             command.extend([
                 '--binary-arg=--use-fake-ui-for-media-stream',
-                '--binary-arg=--use-fake-device-for-media-stream'
+                '--binary-arg=--use-fake-device-for-media-stream',
```

Comment:
In general I wish we would put these flags in wptrunner directly rather than having special configuration here.

Reply:
Okay. That feature needs a bit more definition before I can implement it, so I've opened gh-10823 to discuss it further.

Reply:
...which is in another repository, so I'll need a URL:

https://github.com/w3c/web-platform-tests/issues/10823